package.json

```diff
@@ -277,10 +278,20 @@
     "redux-thunk": "^2.3.0",
     "source-map-support": "^0.5.9"
   },
+  "resolutions": {
+    "@octokit/rest":"16.33.1"
+  },
   "devEngines": {
     "node": ">=7.x",
     "npm": ">=4.x",
     "yarn": ">=0.21.3"
   },
-  "browserslist": "electron 1.6"
+  "browserslist": "electron 1.6",
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged",
+      "pre-push": "yarn test",
```

Comment:
dyt we can add check-deps to the pre-push

Reply:
Yes we can but maybe we have to remove it again